app/assets/stylesheets/_animations.scss

```diff
@@ -0,0 +1,19 @@
+.scaleIn-animation {
+  transform: scale(0);
+  animation-name: scaleIn;
+  animation-duration: 0.3s;
+  animation-fill-mode: forwards;
+  animation-iteration-count: 1;
+}
+
+@keyframes scaleIn{
```

Comment:
Since you're using a cubic for the panel heading you could use it here too instead of using an animation. I think what you're trying to do can be achievable with the [easeOutBack](http://easings.net/#easeOutBack).